app/backend/src/tests/test_api.py

```diff
@@ -182,13 +189,122 @@ def test_update_profile(db):
         assert user.about_place == "My place"
         assert user.hosting_status == api_pb2.HOSTING_STATUS_CAN_HOST
         assert user.meetup_status == api_pb2.MEETUP_STATUS_WANTS_TO_MEETUP
-        assert "Binary" in user.languages
-        assert "English" in user.languages
         assert user.additional_information == "I <3 Couchers"
         assert "UK" in user.countries_visited
         assert "Aus" in user.countries_visited
         assert "UK" in user.countries_lived
         assert "Aus" in user.countries_lived
+        assert user.language_abilities[0].code == "eng"
+        assert user.language_abilities[0].fluency == api_pb2.LanguageAbility.Fluency.FLUENCY_NATIVE
+
+    return user, token
+
+
+def test_language_abilities(db):
```

Comment:
I feel like we should also test for a second language to make sure the repetition works properly?